ts-defs/index.d.ts

```diff
@@ -1636,6 +1658,8 @@ interface Runner {
      */
     screenshots(path: string, takeOnFails?: boolean, pathPattern?: string): this;
 
+    screenshots(options: ScreenshotsOptions): this;
```

Comment:
Remove the declaration with several arguments to people used a new one.

Reply:
This change will lead to BC, will not?

Reply:
Yes, it's my mistake.